share/revng/test/configuration/revng-qa/compile.yml

```diff
@@ -1,27 +1,29 @@
 tags:
-  - name: clang
+  - name: clang-only
   - name: simple-executable
   - name: stripped
   - name: with-debug-info
 commands:
   - type: revng-qa.compiled-with-debug-info
     from:
       - type: source
-        filter: linux and simple-executable and !clang
+        filter: linux and simple-executable and !clang-only
     command: ${TRIPLE}gcc $INPUT -o $OUTPUT $GCC_CFLAGS $COMMON_CFLAGS
   - type: revng-qa.compiled-with-debug-info
     from:
       - type: source
-        filter: linux and simple-executable and clang
-    command: ${TRIPLE}clang $INPUT -o $OUTPUT $GCC_CFLAGS $COMMON_CFLAGS
+        filter: linux and simple-executable and clang-only
+    command: clang $INPUT -o $OUTPUT $GCC_CFLAGS $COMMON_CFLAGS $CLANG_CFLAGS
```

Comment:
`${TRIPLE}clang` would be the right thing to do, I think.

Reply:
I agree, but I that seems non-transparent to me, and it's a lot easier to reproduce when you can just do `any-clang --target=X`.

That being said, now that this finally works, I don't mind any tweaks as long as it keeps working :D

Reply:
I'd prefer `${TRIPLE}clang`. It's the convention for cross-compiling.

Reply:
Restored.